eZ/Bundle/EzPublishCoreBundle/EventListener/RequestEventListener.php

```diff
@@ -82,6 +82,9 @@ public function onKernelRequestForward(GetResponseEvent $event)
                     $request->getContent()
                 );
                 $forwardRequest->attributes->add($request->attributes->all());
+                if ($request->headers->has('X-User-Hash')) {
+                    $forwardRequest->headers->set('X-User-Hash', $request->headers->get('X-User-Hash'));
+                }
```

Comment:
It's hardcoded here too https://github.com/ezsystems/ezpublish-kernel/blob/451c4f92d2ea27200f3f6e135733aa3177c08f89/eZ/Bundle/EzPublishCoreBundle/HttpCache.php#L112